StringPermutation.java

```diff
@@ -0,0 +1,69 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by ioana-chirca on 16-May-17.
+ */
+public class StringPermutation {
+    public boolean isPermutation(String first, String second) {
```

Comment:
This method can be static, it does not use any information specific to any instance of class StringPermutation.